overlord/devicestate/devicemgr.go

```diff
@@ -181,6 +182,7 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	runner.AddHandler("request-serial", m.doRequestSerial, nil)
 	runner.AddHandler("mark-preseeded", m.doMarkPreseeded, nil)
 	runner.AddHandler("mark-seeded", m.doMarkSeeded, nil)
+	runner.AddHandler("prepare-ubuntu-save", m.doPrepareUbuntuSave, nil)
```

Comment:
if we keep this, it should be s/prepare/setup/

Reply:
Good point, fixed!